core/src/main/web/app/utils/basic/commonui.js

```diff
@@ -83,13 +83,15 @@
 
         var showDropdown = function() {
           var togglePosition = toggle.offset();
+          var $notebook = $('bk-notebook');
```

Comment:
you could walk the parent scope up until you find the notebook directive and get the element that way... that might be nice